files/zh-cn/web/css/blend-mode/index.md

```diff
@@ -15,47 +15,328 @@ slug: Web/CSS/blend-mode
 ### 属性值
 
 - `normal`
-  - : 最终颜色永远是顶层颜色，无论底层颜色是什么。其效果类似于两张不透明的纸重叠（overlapping）在一起。
+  - : 最终颜色永远是顶层颜色，无论底层颜色是什么。
+      其效果类似于两张不透明的纸重叠（overlapping）在一起。
+
+      ```html hidden
+      <div id="div"></div>
+      ```
+
+      ```css hidden
+      #div {
+        width: 300px;
+        height: 300px;
+        background: url('https://mdn.mozillademos.org/files/8543/br.png'),
+                    url('https://mdn.mozillademos.org/files/8545/tr.png');
+        background-blend-mode: normal;
+      }
+      ```
+
+      {{ EmbedLiveSample('normal_example', "300", "300") }}
+
```

Comment:
This macro will not render this code sample correctly. But it's ok to keep it. There are a lot of Live Sample should be updated for being converted to markdown.